packages/aws-cdk/README.md

```diff
@@ -560,6 +561,114 @@ This feature currently has the following limitations:
   bucket). Requires version 12 of the bootstrap stack, for the added
   IAM permissions to the `deploy-role`.
 
+
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ 
+
+CDK migrate is currently experimental and may have breaking changes in the future. 
+
+Generates a CDK application from an existing JSON/YAML CloudFormation template.
```

Comment:
Given that this and the sentences below are in paragraph form, not bullet form, these should be complete sentences.